src/app/chat/chat-room/chat-room.component.ts

```diff
@@ -39,6 +39,9 @@ export class ChatRoomComponent extends RouterEnter {
   loadingMesageSend = false;
   isTyping = false;
   typingMessage: string;
+  // this use to show/hide bottom section of text field which have attachment buttons and send button,
+  // when user typing text messages
+  textFieldFocused = false;
```

Comment:
If this flag is only used to show/hide the buttons, it's better to name it something like "hideAttachmentButtons" so that it is self-explained. Otherwise "textFieldFocused" is not a good name to explain the situation